### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by aiorhiroki**
# Release 2.0  🎉 

- Optuna
- Albumentation
- Github Action
- Dataset, DataLoader
- add tversky loss, pretrainable segmentation model and more ...

### CODE REVIEW

File: farmer/domain/model/trainer_model.py [hunk 1]
```diff
@@ -38,6 +39,15 @@ class Trainer(Config, ImageLoader, Optuna):
     cosine_lr_max: int = 0.01
     cosine_lr_min: int = 0.001
     optuna: bool = False
+    seed: int = 1
+    n_trials: int = 10
+    timeout: int = None
+    trial_number: int = None
+    trial_params: dict = None
+    train_params: dict = None
+    optuna_params: dict = None
```
Comment (yumion) [inline farmer/domain/model/trainer_model.py:48]:
```
    trial_number: int = None
    trial_params: dict = None
    train_params: dict = None
    optuna_params: dict = None
```

ここ命名どうにかなりませんか？
ちょっとわかりにくいです

Comment (aiorhiroki) [reply farmer/domain/model/trainer_model.py:48]: 知ってた！
そこは、次のリリースで！
